cli/create.go

```diff
@@ -267,6 +267,20 @@ func createSandbox(ctx context.Context, ociSpec oci.CompatOCISpec, runtimeConfig
 		return vc.Process{}, err
 	}
 
+	// Important to create the network namespace before the sandbox is
+	// created, because it is not responsible for the creation of the
+	// netns if it does not exist.
+	if err := setupNetworkNamespace(&sandboxConfig.NetworkConfig); err != nil {
+		return vc.Process{}, err
+	}
+
+	// Execute prestart OCI hooks
+	if err := enterNetNS(sandboxConfig.NetworkConfig.NetNSPath, func() error {
```

Comment:
This is a bit of a mind-bender :) I think it would be clearer if you assigned to `err` and then checked `err` on a separate line.

Reply:
Ok fair enough ;)

Reply:
done